### PR TITLE
REDVM-495: updating docs for leaf cert rotations.

### PR DESCRIPTION
**Pull request by ashrivastav13**
Added step is workaround to trust newly generated certificate adding it to bosh trusted text box section.

### CODE REVIEW

File: rotating-ca.html.md.erb [hunk 1]
```diff
@@ -7,3 +7,25 @@ This topic tells you how to access BOSH CredHub, check expiration dates,
 and rotate certificates when using <%= vars.product_full %>.
 
 <%= partial vars.path_to_partials + '/certs/rotating-certs' %>
+
+## Workaround step to rotate deployment's certificates using Maestro
+
+The existing versions of the Redis tile for Tanzu Application Service do not support the above [recommended certificate rotation procedure](https://docs.vmware.com/en/VMware-Tanzu-Operations-Manager/3.0/vmware-tanzu-ops-manager/security-pcf-infrastructure-advanced-certificate-rotation.html#services-rotation) for Tanzu Application Service tiles (which uses CredHub Maestro).
```
Comment (pnikonowicz) [inline rotating-ca.html.md.erb:13]: i was unaware of this. we currently have maestro tests that verify that we can rotate certs using maestro. wouldn't this mean that we do support rotating certs using maestro?

**Review by pnikonowicz — CHANGES_REQUESTED** [review]
further discussion needed in regards to workaround. redis shouldn't have any special steps with it's rotation procedure as far as i know.

File: rotating-ca.html.md.erb (hunk 1 above)
Comment (ashrivastav13) [reply rotating-ca.html.md.erb:13]: Okay, we can close this PR.

Comment (pnikonowicz) [reply rotating-ca.html.md.erb:13]: is this the work around for the drain script issue with HA deployments?

Comment (ashrivastav13) [reply rotating-ca.html.md.erb:13]: Yes, it is workaround for drain script

**Review by pnikonowicz — APPROVED** [review]

**Review by apurvasaraiya — APPROVED** [review]

**Pull request merged**